util/loaders/SoldierLoader.ts

```diff
@@ -36,6 +36,7 @@ export class SoldierLoader extends Loader<Soldier[]> {
         baseHp: +this.getSoldierRowValue(rowCounter, scm.baseHp),
         baseMdef: +this.getSoldierRowValue(rowCounter, scm.baseMdef),
         trainingSkill: null,
+        moveType: this.getSoldierRowValue(rowCounter, scm.moveType),
```

Comment:
This has let me see that what I did here needs some improvement. Every change to the col-defs object forces another change in the mapper. I'll make this code auto generate off the column defs I think. Should reduce the busy work.

Reply:
hm.. not sure what you saw here, lets discuss on discord if you want me to do it.